### PR TITLE
Select N1QL Query Swift

### PR DESCRIPTION
**Pull request by jayahariv**
* select query addition with a basic sanity unit test

### CODE REVIEW

**Comment by jayahariv** [comment]
CI failure due to https://github.com/couchbase/couchbase-lite-ios/pull/2854

**Comment by pasin** [comment]
Can you address if the PR validation fail related to the PR or not?

**Review by pasin — COMMENTED** [review]
Can you address if the PR validation fail related to the PR or not?

**Review by pasin — APPROVED** [review]

File: Swift/Database.swift [hunk 1]
```diff
@@ -400,6 +400,15 @@ public final class Database {
     /// Log object used for configuring console, file, and custom logger.
     public static let log = Log()
     
+    /// Creates a Query object from the given N1QL query string.
```
Comment (pasin) [inline Swift/Database.swift:403]: Can you remove the word N1QL?

**Review by pasin — APPROVED** [review]

File: Swift/Database.swift (hunk 1 above)
Comment (jayahariv) [inline Swift/Database.swift:403]:
```suggestion
    /// Creates a Query object from the given query string.
```

File: Swift/Query.swift [hunk 2]
```diff
@@ -150,6 +150,14 @@ public class Query {
     init(database: Database, JSONRepresentation json: Data) {
         queryImpl = CBLQuery(database: database._impl, jsonRepresentation: json)
     }
+    
+    /// Creates a query, given the N1QL string, as from the expression property.
```
Comment (jayahariv) [inline Swift/Query.swift:154]:
```suggestion
    /// Creates a query, given the query string, as from the expression property.
```

**Pull request merged**